docs/Reference/TypeScript.md

```diff
@@ -862,9 +862,8 @@ a more detailed http server walkthrough.
    import path from 'path'
    import fastify from 'fastify'
    ```
-2. Follow the steps in this official [Node.js https server
-   guide](https://nodejs.org/en/knowledge/HTTP/servers/how-to-create-a-HTTPS-server/)
-   to create the `key.pem` and `cert.pem` files
+2. Follow the official documentation of [Node.js in the https section ](https://nodejs.org/api/https.html#httpscreateserveroptions-requestlistener)
```

Comment:
It doesn't seems correct.

https://web.archive.org/web/20201217160232/https://nodejs.org/en/knowledge/HTTP/servers/how-to-create-a-HTTPS-server/

You should provide the `openssl` comment.

Reply:
Okay then I'll add this link: 
https://web.archive.org/web/20201217160232/https://nodejs.org/en/knowledge/HTTP/servers/how-to-create-a-HTTPS-server/

 Is i leave the text as it was before

Reply:
Just copy the commend in the link and write inside the doc.

Reply:
Updated description see here: https://github.com/fastify/fastify/pull/5389/commits/4ec3b6c30637603b4fe2b5321170fe9f7070c21a

Reply:
Lint Docs / Lint Markdown fails

Reply:
now it is ok